pages/blog/automated-releases.md

```diff
@@ -194,7 +194,9 @@ To share the output, you must assign an `id` to the step and declare a variable
 ```yaml
 - name: Get version from package.json after release step
   id: extractver
-  run: echo "::set-output name=version::$(npm run get-version --silent)"
+  run: |
+    npm run get-version --silent
+    echo "version=$(npm run get-version --silent)" >> $GITHUB_ENV
```

Comment:
Nicely done @SumantxD 
Was just wondering if that should be ```$GITHUB_ENV``` or ```$GITHUB_OUTPUT```. Maybe the later one should be there.